content/format-drive.md

```diff
@@ -36,6 +42,6 @@ In this screenshot the drive is a USB Flash Drive, though the concept is the sam
 
 ![Disks](/images/format-drive/disks-format-disk.png)
 
-Now select the **Format Disk...** option then make sure to have the 'Compatible with modern systems and hard drives > 2TB (GPT)' selected (you can use either the Quick option or the Slow option).
+Now select the `Format Disk` option then make sure to have the 'Compatible with modern systems and hard drives > 2TB (GPT)' selected (you can use either the Quick option or the Slow option).
```

Comment:
Is there a reason to change the style here? Do we have an updated style somewhere?

Reply:
This is correct use of the Markdown inline code element. Using bold for word-for-word references to anything is not a standard. I agree with this change.

That said, I would still include the ellipses within the inline code element since the option we're referring to verbatim has an ellipses.

Reply:
It does not follow the style in other articles from my understanding.

Reply:
We have been moving to `Button` for UI elements, so this is correct.